app/bundles/EmailBundle/Swiftmailer/Transport/AmazonApiTransport.php

```diff
@@ -375,6 +375,10 @@ public function getAmazonMessage(\Swift_Mime_SimpleMessage $message)
                 // Reset the parts of the email that has tokens
                 $this->message->setSubject($emailSubject);
                 $this->message->setBody($emailBody);
+                $message_headers= $this->message->getHeaders();
```

Comment:
I think this is correct but I need to confirm it, these lines of code must be added as well to line #363 so we include both cases queue and immediate send.

Reply:
i checked queue without adding those lines on #363 and it looks fine to me

Reply:
That is correct, the queue headers are embedded as serialized values to the sent object so we do not need them